src/style/variables.scss

```diff
@@ -82,5 +82,8 @@ $l-dark-red: #8C4646;
 $l-green: #73BF76;
 $l-blue: #588799;
 
+$thumbs-up: #006144;
```

Comment:
Colors are quite similar to the default and it's hard to spot whether icon is active or not. It might be due to mi color blindness though.